foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiStreamingHandler.java

```diff
@@ -42,20 +36,8 @@ Stream<D> handleResponse(@Nonnull final ClassicHttpResponse response)
   @SuppressWarnings("PMD.CloseResource")
   private Stream<D> parseResponse(@Nonnull final ClassicHttpResponse response)
```

Comment:
> The stream is closed by the user of the Stream

This is false now. Could you update this comment and duplicate it in `IterableStreamConverter.lines()`
Alternatively just merge `parseResponse` inside of `handleResponse` since `parseResponse` only contains a `return` and nothing else